docker/Dockerfile.olbase

```
@@ -29,6 +29,9 @@ RUN apt-get -qq update && apt-get install -y \
     python-psycopg2 \
     python-yaml
 
+# Update pip to at least 9.0
+RUN pip install --upgrade pip==9
```

The version on ol-dev0 and ol-web3 is 19.3.1

```suggestion
RUN pip install --upgrade pip==19.3.1
```

---

Why not `pip==20.0.1`?

---

That's what's on our production environment. We don't have a great way of updating things there yet (they're not running docker, so we have to update each dep manually, which is a time sink for a 2.5 person team). I think keeping things in sync is more beneficial than keeping things bleeding edge; once we have docker in more places, updating dependencies will be much easier.